CODE_OF_CONDUCT.md

```diff
@@ -2,7 +2,7 @@
 The community has elected to adopt the Contributor Convenant in order to foster an open and welcoming environment where contributors can feel safe to make contributions in any way they wish. 
 
 The full text of the code of conduct is below. Instances of abusive, harassing, or otherwise unacceptable behaviour may be
-reported by contacting one of our code of conduct representatives, David Moulton from eLife (d.moulton@elifesciences.org) or Girish Nair from Hindawi (girish.nair@hindawi.com).
+reported by contacting one of our code of conduct representatives, David Moulton from eLife (d.moulton@elifesciences.org).
```

Comment:
Should "reported by contacting one of our code of conduct representatives" be changed to "reported by contacting our code of conduct representative" until a replacement for Girish is found?

Reply:
Good spot, I've updated